src/lib/components/Navbar.svelte

```diff
@@ -61,7 +67,7 @@
   };
 </script>
 
-{#if activePromotion}
+{#if activePromotion && isEnabledMermaidChartLinks}
```

Comment:
Please move this check into the `getActivePromotion` function, so we could consolidate the promotion display logic.